src/gis/supported_formats.md

```diff
@@ -0,0 +1,218 @@
+# Supported Formats
+<!-- reference -->
+  
+[[toc]]
+
+## Vector data 
+
+
+| Format         | <MobileAppName /> (Android, iOS)  | QGIS (Windows, macOS, Linux)  | Note                                                                  |
+|----------------|-----------------------------------|-------------------------------|-----------------------------------------------------------------------|
+| GeoPackage     | :heart_eyes:                      | :heart_eyes:                  | Collaborative editing via <GitHubRepo id="lutraconsulting/geodiff" /> |
+| Shapefile      | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          |
+| Delimited text | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          | 
+| Virtual layer  | :warning:                         | :warning:                     | Not supported by <GitHubRepo id="lutraconsulting/geodiff" />          |                 
```

Comment:
not supported by geodiff probably won't tell much to ordinary users - how about something like "automatic resolution of edit conflicts is not supported" / "automatic merging is not supported" ? probably some better wording is needed...